### PR TITLE
r-sahu recipe added

### PR DESCRIPTION
**Pull request by sk-sahu**
Checklist

- [x] Source is from official source
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dbast** [comment]
@sk-sahu Thanks for creating this recipe. Please use https://github.com/bgruening/conda_r_skeleton_helper the generate/update this recipe (also ensure to have the newest conda-build version installed).

**Comment by sk-sahu** [comment]
Hi @dbast I have made the correction. Its passed the checks. Can you please merge the PR.

**Comment by dbast** [comment]
This is:
* a package not on cran
* just your personal stuff as the description tells: `Personal R function library`
* it is using a git_url instead of a tarball
* lots of more findings (all the commented lines, noarch, outdated lines in build files etc)

Having this on conda-forge does not seem feasible.

Why don't you just upload it to your own (free) anaconda.org channel as noarch recipe?

**Comment by cbrueffer** [comment]
Agreed, this seems more appropriate for a personal channel than conda-forge.

File: recipes/r-sahu/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sahu
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  git_url: https://github.com/sk-sahu/sahu
```
Comment (bgruening) [inline recipes/r-sahu/meta.yaml]: can you please use a tarball?

Comment (sk-sahu) [reply recipes/r-sahu/meta.yaml]: Hi @bgruening I added the tarbal link in url. In this case its failing for windows, Tried to check log but couldn't figure out. With git_url it was passing all platforms. Should I do [this skipping platform](https://github.com/conda-forge/staged-recipes#3-how-do-i-exclude-a-platform)

**Pull request closed without merging**